install.rdf

```diff
@@ -5,20 +5,19 @@
     <em:bootstrap>true</em:bootstrap>
     <em:type>2</em:type>
     <em:unpack>false</em:unpack>
-    <em:version>1.0.0</em:version>
+    <em:version>0.1.2</em:version>
```

Comment:
One of the warts of this release process: AMO will refuse to sign a version number equal to or lower than a version that has been previously submitted.